accelerator/core/response/ta_send_transfer.c

```diff
@@ -11,6 +11,7 @@
 ta_send_transfer_res_t* ta_send_transfer_res_new() {
   ta_send_transfer_res_t* res = (ta_send_transfer_res_t*)malloc(sizeof(ta_send_transfer_res_t));
   res->hash = NULL;
+  memset(res->uuid, 0, UUID_STR_LEN);
```

Comment:
Is it for checking whether there is a UUID by checking  `res->uuid[0] == 0`?

Reply:
It is